code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -266,7 +267,7 @@
 	switched_on_qualities = list(QUALITY_CUTTING = 30, QUALITY_SAWING = 30)
 	switched_off_qualities = list(QUALITY_CUTTING = 10, QUALITY_SAWING = 10)
 	tool_qualities = list(QUALITY_CUTTING = 10, QUALITY_SAWING = 10)
-	active_time = 50
+	active_time = 100
```

Comment:
Time is in deciseconds, which is why leaving it unmodified is bad. 10 SECONDS would be better (the all caps is important)